www/docs/en/dev/guide/platforms/electron/index.md

```diff
@@ -52,7 +52,7 @@ Electron is a framework that uses web technologies (HTML, CSS, and JS) to build
 
 * **PowerShell**, for Windows 7 users, must be at version 3.0 or greater for [app signing](https://www.electron.build/code-signing#windows).
 
-* **Debugging Tools** for Windows of Windows SDK 10.0.15063.468, if you plan on creating a full distribution.
+* **Debugging Tools**, for Windows of Windows SDK 10.0.15063.468, if you plan on creating a full distribution.
```

Comment:
"for Windows of Windows" doesn't sound right

Reply:
Yeah, maybe that's better.